docs/enhancement-proposals/off-ep-0009.md

```diff
@@ -0,0 +1,88 @@
+# OFF-EP 0009 — Add LJPME
+
+**Status:** Submitted
+
+**Authors:** Matt Thompson, John Chodera
+
+**Acceptance criteria:** Unanimity (4 approving reviews) or partial support (2 approvals and 2 week period with no reviews requesting changes)[https://openforcefield.atlassian.net/wiki/spaces/MEET/pages/2638774273/09-05-23+SMIRNOFF+Committee+Meeting]
+
+**Stakeholders:**
+
+**Created:** 2023-09-05
+
+**Discussion:** [PR #40](https://github.com/openforcefield/standards/pull/40), [PR #44](https://github.com/openforcefield/standards/pull/44), [Issue #11](https://github.com/openforcefield/standards/issues/11), [Toolkit issue #989](https://github.com/openforcefield/openff-toolkit/issues/989#issuecomment-862792421), [OFF-EP-0008](https://github.com/openforcefield/standards/pull/53#issuecomment-1661316600)
+
+**Implementation:** [``openff-standards``](https://github.com/openforcefield/openff-standards)
+
+## Abstract
+
+This change adds `periodic_method="Ewald3D"` as a supported attribute value in the `<vdW>` tag.
+
+## Motivation and Scope
+
+There are compelling reasons for force fields to handle long-range vdW interactions with an Ewald sum (so-called LJPME, used here synonymously) analogously to how electrostatics are conventionally handled, despite the increased computational cost. While tail corrections largely mediate the errors introduced by cutting off vdW interactions before they naturally decay to zero, they do so imperfectly, particularly in nonisotropic systems. These differences, even if small, also imply that a force field fitted with cut-off vdW interactions would not likely perform as well if LJPME is used.
```

Comment:
I really like how this section is worded -- comprehensive but to the point.

Reply:
One small add (optional) could be to add one more sentence saying, "Additionally, without LJPME, some observables may have substantially different values depending on the user-selected cutoff."